src/components/Button.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import styled from 'styled-components'
+import buttonBG from '../assets/hc_button.svg'
+import { hexToRgba } from '../utility/utilities'
+
+const buttonSize = {
+  small: '60px',
+  default: '80px',
+  large: '300px',
+}
+
+const buttonColors = ['primary', 'secondary', 'tertiary']
+
+const StyledButton = styled.button`
+  display: block;
+  text-decoration: none;
+  font-family: 'HK Grotesk';
+  font-weight: ${p => p.theme.typography.h2.weight};
+  border: transparent;
+  transition: all 250ms;
+  min-width: ${p => (p.size ? buttonSize[p.size] : buttonSize['default'])};
+  text-align: center;
+  padding: 0.75em;
+  border-radius: 3px;
+  margin: 1em;
+  :hover {
+    cursor: pointer;
+  }
+  :focus {
+    outline: 0;
+  }
+  :disabled {
+    cursor: not-allowed;
+    opacity: ${p => p.theme.opacity.disabled};
+  }
+  ${p =>
+    (p.color === buttonColors[0] || !buttonColors.includes(p.color)) && // primary color (gradient button) or not specified color
+    `color: ${p.theme.colors.background};
+    background: ${p.theme.colors.primaryGradient};
+    :hover {
+      background: ${p.theme.colors.primaryGradientHover};
+    }
+    :disabled {
+      background: ${p.theme.colors.primaryGradient};
+    }
+    ${
+      p.theme.custom_imgs === 'hc' &&
```

Comment:
hmM I know you didn't create this but maybe we should change this to `themeName` instead of `custom_imgs`, since we aren't really using it for an image but we are using it to check which theme we are on.
If we actually want to use `custom_imgs` maybe we should export the url to the asset from the themeprovider instead of importing the asset. But only button uses the background image so i dont think its a big deal.

Reply:
i think i'll leave this for now bc it's @jackyzha0's work here and i'm not 100% sure what the expectations for themes would be, going forward? is hc the only event with picture buttons?